sentry-android-core/src/main/java/io/sentry/android/core/Installation.java

```diff
@@ -12,7 +12,7 @@
 import org.jetbrains.annotations.Nullable;
 import org.jetbrains.annotations.TestOnly;
 
-final class Installation {
+public final class Installation {
```

Comment:
why making this class public? Do we want to expose it to the users?

Reply:
Oh that's an oversight due to some refactoring, let me revert that 👍